IDEAS/Buildings/Data/Constructions/FloorOnGround.mo

```diff
@@ -1,18 +1,23 @@
 within IDEAS.Buildings.Data.Constructions;
 record FloorOnGround "Floor on ground for floor heating system"
   extends IDEAS.Buildings.Data.Interfaces.Construction(
-    locGain={3},
+    locGain={2},
     incLastLay = IDEAS.Types.Tilt.Floor,
     mats={
       IDEAS.Buildings.Data.Materials.Concrete(d=0.20),
-      IDEAS.Buildings.Data.Insulation.Rockwool(d=0.1),
+      IDEAS.Buildings.Data.Insulation.Eps(d=0.1),
       IDEAS.Buildings.Data.Materials.Screed(d=0.08),
-      IDEAS.Buildings.Data.Materials.Concrete(d=0.015)});
+      IDEAS.Buildings.Data.Materials.Tile(d=0.015)});
```

Comment:
I would change this value to 0.02 (in correspondance to EPC-based construction layers in #1420)